README.md

```diff
@@ -6,8 +6,7 @@ install Python 3.4
 install pip
 
 pip install virtualenv
-virtualenv -p /usr/bin/python3.4 virtualenv
-(måske skal stien til python rettes)
+virtualenv -p $(which python) virtualenv
```

Comment:
Yep: it seems that 3.x.x is required, probably because of "InsecurePlatformWarning" when using a 2.x.x